diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -157,22 +157,37 @@ open class DiktatJavaExecTaskBase @Inject constructor(
         // appending the flag with the reporter
         setReporter(diktatExtension, flag)
 
-        if (diktatExtension.output.isNotEmpty()) {
-            flag.append(",output=${diktatExtension.output}")
+        val outFlag = when {
+            // githubActions should have higher priority than a custom input
+            diktatExtension.githubActions -> ",output=${project.projectDir}/${project.name}"
```

Comment:
```suggestion
            diktatExtension.githubActions -> ",output=${project.buildDir}/${project.name}"
```
I think it's better to keep repo clean

Reply:
By for now, let's create grabage. As it happens in separate action - it should not affect anyone. But I feel more safe :)